docs/elements.md

```diff
@@ -516,16 +516,16 @@ def generate(data):
   z = x + y + 1
 
   # Answer to fill in the blank input stored as JSON.
-  data['correct_answer']['symbolic_math'] = pl.to_json(z)
+  data['correct_answers']['symbolic_math'] = pl.to_json(z)
 ```
 
 #### Customizations
 
 Attribute | Type | Default | Description
 --- | --- | --- | ---
-`answers-name` | string | — | Variable name to store data in. If the correct answer `ans` is a `sympy` object, you should use `import prairielearn as pl` and `data['correct_answer'][answers-name] = pl.to_json(ans)`.
+`answers-name` | string | — | Variable name to store data in. If the correct answer `ans` is a `sympy` object, you should use `import prairielearn as pl` and `data['correct_answers'][answers-name] = pl.to_json(ans)`.
 `weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
-`correct-answer` | float | special | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.
```

Comment:
Same as above.

Reply:
Hmm.

I get an error if I use `correct-answer` for symbolic input.

<img width="849" alt="Screen Shot 2021-05-08 at 2 13 23 PM" src="https://user-images.githubusercontent.com/2507459/117553660-a8e87b80-b007-11eb-8fd6-050342b9c55f.png">

Reply:
There are two things you might be confusing here. The Python way to indicate a correct answer is `data["correct_answers"][field] = value`, but you can also set it in the HTML element itself:
`<pl-symbolic-input answers-name="fieldname" correct-answer="x+y^2" ...></pl-symbolic-input>`

Reply:
Ohhh I see!! Thank you.

I think that is very confusing. Am I wrong about this? The parameter names should be the same shouldn't they? I suppose that would be a much bigger fix...

Reply:
The `correct-answer` parameter to the HTML element is the answer to that particular element. The `correct_answers` dict on Python is a dictionary of all the correct answers for all the elements.

Reply:
Ah and you can have multiple elements, but in a python dict it's a list of all of them.

Got it. Thanks, I'll revert the changes on these